x-pack/plugins/security_solution/public/detections/components/alerts_table/translations.ts

```diff
@@ -281,13 +281,6 @@ export const ADD_TO_NEW_CASE = i18n.translate(
   }
 );
 
-export const ADD_TO_CASE_DISABLED = i18n.translate(
```

Comment:
The translation is used in Security Solution.

Reply:
Do you mean "not used"?

I didn't see it being used anywhere.

Maybe you can also delete the occurrence in x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/hooks/translations.ts.

Reply:
Yes sorry. I meant they are not used. Done https://github.com/elastic/kibana/pull/156828/commits/700829f7013f9acf0c1894cf9f2b545eea1a89bb